concepts/observability-strategy/configurable-monitoring/assets/monitoring-future.drawio.svg

```diff
@@ -1,14 +1,14 @@
1/iaWHxvIxyx7/zG4P8MSFj0r6+bt1XSPmoabxmmpKfdQuOmFeqSSSjCzInKUru2/QbodO8j1NQmp5fCFkVDW7R8H9M6bZwt5yG1pQU5ELnm16Nqos7AbPZzJtOi9a2wtftUTAJYFBeY71cNTc9R8X1b/Ampn+1rsF/fy1v+cqDNelu03qMgrBtE1KmwX9Vo2BD+tqemBPkaQpKe56G4ZzDnQRaN9WybKyVdc9wgmj8bJ6L2WBNnrdGtaa7zjK01TqvSJzS3KxKj7ytGLFpBgDYKJ6zbd0MGJtvJyfrbIrruTpeBrrQPB1F2RzTF6crfkqckxsq82gTZaMVE6jWXYISw7MSlwQUWIxpJmg6ioAJ7z3g+zCyg/d+oOC9IXRxgQnvoQ28N8WFH3j/gfcfeP+B9z8Y3kcQjyLfhPcjbwKKNawFvK8THgLvdbivu7Th3uttpQo1LjagZIakBpBKPitgVHK3AaIKhiTel2BRDi+BooKJGiSatrut+FGCQzVDCQt8WAgHDSq0utYD1WxFCXUGSVcgt0PSpTntYKcrsleFoezqOCo7FgC1S3Nq5JFyXgY1KIZWOFN3EAhTz1zZrlBIVwlA3Cpx99Cxvw8VDSxvwDy41mIxWFFqCawUva45cryqBxej6uepseG70FgRTl+EyoYXo7I1OrcB+auExxeEztB9F7ruhc7u/q63q/+BtjE6rW0IYbZWr4TiZMhWqAmeUr56sRgfdtBLpcZhMy/sKTWOUaDFja5niBuDvsJGV69wXK9WSTxl1kBSq5zvkInpkfNDNWI3lD1cU3HJRoJmpx9usf7zdokY5WmbU7wUq6VJJhZKv+ZMKOxa98+MDzfrnNv1ii3bnRvOULt20qFO0qu0oAKYBmmZ6if9GcpIl9boaYewrh9/fcLZMxfLw+f1BCeYvisBGQTin1IgXvCS+dwmayYS3RzYo1GZpTLrUpJihc8VCSXxPGU/p4xhbGJwwxnFMDK5rhqWcRQVQZtJhrKU+xSKL2Pc0DOUHU1C8noTkhnjhsJonP+gJa6Sxz+atGAoYxyDGF1a45NKS8e4/+XckH5kMQF15Qjf2KbAWJPSpwzNUIqs+pfZDAcvBMrheGKLu/5INgJg4K7JBnqLyvyxxsUTLtXF93Kp3nWlLudRD0yjvqZAuneh3tP6W1/COoqRivhRTCEVPOyvYaG+hq2C9SYsNBRhlULpaLUpK6RKrfQxI5vtK42bXanSvtG5xZLj/SszU+m8NwgQ26cuK1vnw1EbB4bOlQND6zm7YvwjzmLGau6073bqgMVEXtALkJw8kafu+lX6B2BX937yePpNeyqYKhPZA1PPsMHWd1YZn2OB1/lVke/IpwzPeDPN1noEylXiC2ILbNmeuoeXGWYQjCbFfFydKyVgk8ObAbwzKvhO4NCgr97wXl1l0N5TboJE52osdt4LVSh/vdJPDoG65lBES2azHPfjIvX1+mMt5z6d24l2gO13Ysb9vr05MX3l/T4Tu26H8OG0iV2oL6M/gsHO4lR3Ur51NBh4lxgNej3XbpvNg+XWwdMEge9jr83eam4w2tX/RFFg4LEYpPUZyzGhw9YX7U/QV4gY6I7sqYwHPec3Nk081aOH8wkJBXq8PiRkC7rAHQey5lkJCoEcadb2aDUm9N4WRhvorPHQ4q4tHQ1bUHwQEh+PotC8rUuGVQ+8BWB6SjUX+HsAUA3pDux/4NYwJXR3A/84xFQ3d9SHi3tYN+sn/H7/48ujZmIWw8IOZpARWq4uwN1wbGvHTKgw1at0aU88aGNh5R1TCzCiT3fweQk43D3AIUu6E4yIct/+HaPWs/hdJSAk29LzW5LO4vmaeejzcPtCSWwkfQAEvqTtdrI+wJMmVUI1O+5dX/NSnODqxtS9MWSFM/S+d/aZivm9be0TEZx8ABAtOcOqEzftX2XY/BivcBKnuG6bSz3FL0V0y20z7jxySNJpfwuSBEotEQSnlOQxYfJelyNazC5HlluvkavJ5QDr3uUFFFQsVDuL1TXOUxXEHSsB4wtx3qFhM1BzynvCYKCmzvwdYfARGA/0oPNPRKeLc/HEwnbO2BPLSZShP+rBFYsbf59vCQoVoxAJljY4B8b1gP+ysDqjs17Dut8wTWZ4zKg3aPoNp5HuM39K4m88XRUhiiIyL4/UR9sU0Yzx/OfXSccGTxXgGLs6T/vKuYM3ORplJcVj3LPVPUV/oryQKbsuthC+cSI9ODCRrvV3XUXVduV5VLeteu1j3b96lNBelgfoi6rHdb4YnFniW5iwlRy354jp6nS0Fc87VIfY8bX6Xlzrebg+smr1u8dqy9MPovWVVRP2cprzfqfZ/RGoa0bDCyFM25htLBl9V2PYRXvQnmrcdj2oUOKTedDOq76jyxehWkXpz7H5+krydp1TsuwLOG3BpvYOAj1S7g023+QNL2dg5uFBdZBOZi40W7JocLYWHSp1xK4WrZ4wq/dS9WDR+sq4z0DI8jmWQxy9wlKDo+8NAfSdMNZ20L7N2S/vSuambzivOoY6N20kHkTu7b3gaXgJiQd4viB7bNgUKAlJ9QXO9jBWaP3lauwpNkLkjOH0mv9fAowwTVCex1NBfoiTulCWRqJTdR6ZUar27nFEqKv4zjO7Z6D3irrC0bHBheIJfRWULSq+XtH4o9lc4NyS5YqJMNXfr3Eeh9MtuEp1d4EodbX8pKnoYeNtk4Ge4rIba7y2nGRIO2q87lxOGhrfNGMsKNkI6YSzaId0hLJ7j4uK0tM1i6Cd3yc5zp7RJE5iym+dv1YmNryK4Yiy0et4FyiHyUID61xD+GYj6xXqW15Kbt2nz3FG0uX54sHuXUh2RTLWd0saS3k23lYR6sBwYfGJHFG7YNQxQpEC6r0vYOCTmI9Qa3u3bcXdJYifvL6ngGtVcesaSQ8Peimtr7zgzVWPMXTOg2o7jpWJjg5sBtVrslvdmxdkg/t/AQ==&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
Pls fix the spelling:
- `k8s` => `Kubernetes`
- Why do we say `Otel Collector` in the document texts, but `otel-collector` here?
- I'd use title case for `Telemetry Operator`.
- `datadog` => `Datadog`
- `dynatrace` => `Dynatrace`
-